server/protocols/jmap-rfc-8621/src/main/scala/org/apache/james/jmap/mail/Email.scala

```diff
@@ -313,18 +314,18 @@ class EmailViewReaderFactory @Inject() (metadataReader: EmailMetadataViewReader,
   }
 }
 
-sealed trait EmailViewReader {
-  def read(ids: Seq[MessageId], request: EmailGetRequest, mailboxSession: MailboxSession): SFlux[EmailView]
+sealed trait EmailViewReader[+EmailView] {
```

Comment:
For my knowledge, what does the `+` before the class name in brackets stands for?

Reply:
It's like `<? extends EmailView>`

Reply:
but better :)
It means that EmailViewReader is covariant on EmailView
Which means that  EmailViewReader[SomethingThatExtendsEmailView] is an EmailViewReader[EmailView].